client/state/jetpack-connect/reducer/test/jetpack-connect-authorize.js

```diff
@@ -33,19 +32,21 @@ describe( '#jetpackConnectAuthorize()', () => {
 
 	test( 'should default to an empty object', () => {
 		const state = jetpackConnectAuthorize( undefined, {} );
-		expect( state ).to.eql( {} );
+		expect( state ).toEqual( {} );
 	} );
 
 	test( 'should set isAuthorizing to true when starting authorization', () => {
 		const state = jetpackConnectAuthorize( undefined, {
 			type: JETPACK_CONNECT_AUTHORIZE,
 		} );
 
-		expect( state ).to.have.property( 'isAuthorizing' ).to.be.true;
-		expect( state ).to.have.property( 'authorizeSuccess' ).to.be.false;
-		expect( state ).to.have.property( 'authorizeError' ).to.be.false;
-		expect( state ).to.have.property( 'isRedirectingToWpAdmin' ).to.be.false;
-		expect( state ).to.have.property( 'autoAuthorize' ).to.be.false;
+		expect( state ).toEqual( {
```

Comment:
The previous version of this test was open for additional properties here. The new one isn't.

I like that we're more strict about the properties that we're expecting, I just wanted to make sure this is done on purpose.

If not, we might want to use `toMatchObject` or `toHaveProperty` instead.

Note: this applies to various other locations in this PR where you update `.to.have.property` calls to use a common `toEqual`.

Reply:
Part of this PR for me was exploring and learning the Jest API 🙂

In most of these cases, I went for the strictest passing version without significantly rewriting the tests. I think it's good for now.

`toMatchObject` or a series of `toHaveProperty` would have been a more precise translation, but I'd vote to leave this, and relax it in the future if that becomes necessary.

The description of this test is `should set isAuthorizing to true when starting authorization`, which isn't what is actually tested, so I'm not too worried about respected the tests as they were.